.husky/pre-commit

```diff
@@ -15,4 +15,5 @@ if [ -z "$changed" ]; then
 fi
 
 npm run generateTelemetry
+npm run validateDependencies
```

Comment:
Can we maybe do this in the linter step for the build? I'm wary of adding yet more stuff to pre-commit. Pre-commit already takes like 60 seconds for me.

Reply:
@rchiodo 👍  I dislike slow commit, 60 seconds is too much. I'll move it to the linter since we always do PR.